bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties

```diff
@@ -244,3 +244,4 @@ config.size.mismatch = [{0}] the size for configurable variable ''{1}'' is expec
 invalid.utf8.byte.array.value = array contains invalid UTF-8 byte value
 regexp.match.string.negative.index = start index cannot be less than 0
 regexp.match.string.index.out.of.range = start index ''{0}'' cannot be greater than input string length ''{1}''
+regexp.empty.character.class = "unsupported empty character class"
```

Comment:
```suggestion
regexp.empty.character.class = "Empty character class disallowed"

```

Reply:
If we use disallowed, then it means that it will not be allow in future.
But we can allow this by introducing `\0` in future ryt?
Shall we go with unsupported.

Reply:
@pcnfernando

Reply:
Had a internal conversation and came to conclusion that disallowed is ok